Dockerfile

```diff
@@ -1,26 +1,23 @@
-#
-# Build
-#
-FROM maven:3.9.5-amazoncorretto-17-al2023@sha256:eeaa7ab572d931f7273fc5cf31429923f172091ae388969e11f42ec6dd817d74 as buildtime
+FROM maven:3.9.3-amazoncorretto-17@sha256:4ab7db7bd5f95e58b0ba1346ff29d6abdd9b73e5fd89c5140edead8b037386ff AS buildtime
+
 WORKDIR /build
 COPY . .
-RUN mvn clean package
 
+RUN mvn clean package -DskipTests
+
+FROM amazoncorretto:17.0.8-alpine3.18@sha256:0c61f12abfb091be48474e836e6802ff3a93e8e038e0460af8c7f447ccbd3901 AS runtime
 
-FROM amazoncorretto:17.0.9-alpine3.18@sha256:df48bf2e183230040890460ddb4359a10aa6c7aad24bd88899482c52053c7e17 as builder
-COPY --from=buildtime /build/target/*.jar application.jar
-RUN java -Djarmode=layertools -jar application.jar extract
+VOLUME /tmp
+WORKDIR /app
 
+COPY --from=buildtime /build/target/*.jar /app/app.jar
+# The agent is enabled at runtime via JAVA_TOOL_OPTIONS.
+ADD https://github.com/microsoft/ApplicationInsights-Java/releases/download/3.4.15/applicationinsights-agent-3.4.15.jar /app/applicationinsights-agent.jar
```

Comment:
Doesn't `afm-calculator` use the ELK stack to handle logs? Beware of conflicts between open-telemetry and app-insight that we encountered with `api-config`